jml-build/sample.local.mk

```diff
@@ -1,12 +1,4 @@
-# to speed up repeated builds, use ccache
-CXX := ccache g++
-CC := ccache gcc
-FC := ccache gfortran
+COMPILER_CACHE:=ccache
 
-# to get colorized output with ccache, put colorccache on your path and use
-# CXX := colorccache g++
-# CC := colorccache gcc
```

Comment:
It is already enabled in `mldb/Makefile`: https://github.com/mldbai/mldb/blob/master/Makefile#L5